doc/src/sgml/installation-meson.sgml

```diff
@@ -387,12 +387,90 @@ meson test
   </sect2>
  </sect1>
 
- <sect1 id="install-getsource-meson">
+ <sect1 id="get-source-meson">
   <title>Getting the Source</title>
 
+   <para>
+   You can download the source code in two ways - via git or the source code
+   distribution tarballs.
+   </para>
+
+  <sect2 id="get-source-git">
+   <title>Getting the Source via <productname>Git</productname></title>
+  <para>
+   With <productname>Git</productname>, you can make a copy of the entire code repository
+   on your local machine, so you will have access to all history and branches
+   offline. This is the fastest and most flexible way to develop or test
+   patches.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     To begin using the Git repository, make a clone of the official mirror:
+
+<programlisting>
+git clone https://git.postgresql.org/git/postgresql.git
+</programlisting>
+
+     This will copy the full repository to your local machine, so it may take
+     a while to complete, especially if you have a slow Internet connection.
+     The files will be placed in a new subdirectory <filename>postgresql</filename> of
+     your current directory.
+    </para>
+
+    <para>
+     The Git mirror can also be reached via the Git protocol. Just change the URL
+     prefix to <literal>git</literal>, as in:
+
+<programlisting>
+git clone git://git.postgresql.org/git/postgresql.git
+</programlisting>
+
+    </para>
+   </step>
+
+   <step>
+    <para>
+     Whenever you want to get the latest updates in the system, <command>cd</command>
+     into the repository, and run:
+
+<programlisting>
+git fetch
+</programlisting>
+    </para>
+   </step>
```

Comment:
Hm, that'll not actually update anything. Do we really want to give a git tutorial?

Reply:
this is mostly copy pasted, but I should have read it in detail. Will replace with`git pull`.

I think having a line on how to clone a repository and how to fetch recent changes is good enough. I think those 2 are probably required in my opinion and more than that will end up becoming a git tutorial.